### PR TITLE
Update faction data for randomized assignment project

### PR DESCRIPTION
**Pull request by sixlettervariables**
This is a work in progress. I'm updating faction data so that assigning a random faction to a person is reasonable enough for MekHQ's purposes.

### CODE REVIEW

File: MekHQ/data/universe/factions.xml [hunk 1]
```diff
@@ -1777,6 +1787,9 @@ tags - a comma-separated list of tags. Currently recognised tags: "is", "periphe
       <nameGenerator>General</nameGenerator>
       <colorRGB>191,171,120</colorRGB>
 		<tags>periphery,minor</tags>
+		 <start>2779</start>
+		 <!-- May be sooner than that, but no later -->
+		 <end>2822</end>
```
Comment (jayof9s) [inline MekHQ/data/universe/factions.xml]: FRR disappeared during the 2nd SW, which lasted 2830 - 2864. End date should fall in that range.

Comment (sixlettervariables) [reply MekHQ/data/universe/factions.xml]: I noticed that, but it seems that they had no planets by 2822. Probably a small inconsistency. I can make it 2830-ish.

Comment (sixlettervariables) [reply MekHQ/data/universe/factions.xml]: Trying to source a date, I found this in The First Succession War (p100), emphasis added:

> In a 2792 naval raid, the Commonwealth attempted to seize Finmark’s shipyard...the spiteful Lyran commander ordered a nuclear bombardment of the planetary surface, targeting all major settlements for destruction. With the death of its capital world and the associated political, industrial, and economic resources, *the FFR collapsed over the next decade*.

Comment (jayof9s) [reply MekHQ/data/universe/factions.xml]: Fair enough, I'm good with the date then.

**Pull request merged**